packages/react-vapor/src/components/filterBox/FilterBox.tsx

```diff
@@ -68,7 +68,7 @@ export class FilterBox extends React.Component<IFilterBoxProps, any> {
     }
 
     componentDidUpdate(prevProps: IFilterBoxProps) {
-        if (this.props.filterText !== prevProps.filterText && this.filterInput?.value !== prevProps.filterText) {
+        if (this.props.filterText !== prevProps.filterText && this.filterInput?.value !== this.props.filterText) {

```

Comment:
Can we simply use `if (this.props.filterText !== prevProps.filterText)` here? I'm not sure if the second condition is necessary :thinking:

Reply:
I think it's required by the FilterBox (not the connected one), you can do some testing if you need but to avoid any unnecessary changes I would keep the proposed conditions